src/Engine/Rendering/ForwardRenderer.cpp

```diff
@@ -51,9 +53,11 @@ void ForwardRenderer::initializeInternal() {
     initShaders();
     initBuffers();
 
-    auto cameraManager = new Scene::DefaultCameraManager();
-    Ra::Engine::RadiumEngine::getInstance()->registerSystem( "DefaultCameraManager",
-                                                             cameraManager );
+    auto cameraManager = static_cast<Ra::Engine::Scene::CameraManager*>(
+        Engine::RadiumEngine::getInstance()->getSystem( "DefaultCameraManager" ) );
+
+    auto comp = cameraManager->getCamera( 0 );
+    comp->getCamera()->resize( m_height, m_width );
```

Comment:
Well, it is strange to do that here. Why to get the camera manager only to resize the camera number 0 ? If this is really needed, why to not doing this in the `resizeInternal` method ?

Reply:
Well, the init pipeline is not very clear. 
The idea is that camera 0 is the default rendering camera, the only one that need to have a viewport size corresponding to the actual viewport. 
Other camera's can keep their loaded size. 
When switching camera's, only data is copied from a given camera to camera 0.

Reply:
OK, so, it should be better to do that on the base class Renderer, and only into the `resize` methods.
`initialize` is called only once when a renderer is added to the viewer (or any object that manages the rendering) and explicitly call the `resize` method with the correct size. 
The base `resize` method from the class Renderer, who call the concrete renderer `resizeInternal` method, is called each time the viewport change. So this is the place where Camera should be resized.
ForwardRenderer is just a renderer among others and should not do things other than just managing its rendering sequence and renderTechnique. All things that should be common to any renderer should be done in the base RendererClass.

Reply:
Also in the current version, setViewport (the new resize name ;)) is called from the Viewer.
Do you think it's better to let the renderer control camera viewport size ?
On my side I think yes, but it recognize the renderer to be in charge of the camera management.

Reply:
I prefer that the renderer just use the given camera and is not in charge of its management. If you can put any camera management method outside of the renderer (even the base class) and just make it to render using the camera (or the associated matrices as done actually), it should be better.

Reply:
But we have to think about something. Do we pass the camera to the "render" method or do we still pass some "ViewingParameters" that are disconected from the camera and contains only the associated matrices (viewMatrix and projMatrix) ?

Reply:
The last point is clearly out of scope of this PR, 

The question is : when/who do resizee the camera ?
The Viewer ?
The Renderer (I mean Ra::Engine::Renderer) ?

Reply:
The viewer